### ISSUE TITLE
Hey I need to talk to you about your project

### ISSUE THREAD

**Issue by X8X-XZDX** [GarrettGunnell/Iterated-Function-Systems#1]
Hello,

This is not about a bug, first off.

Second off, this is amazing work.

I had a small list of questions and wanted to know if we could communicate on discord.

https://discord.gg/6wSz8H6p

This is my discord server, sort of a pre-stage community before things really start kicking off.  My current project is related to AI, and I think your draw engine here perfectly simulates what an idea looks like.  

If you would be willing to chat sometime soon, I would really appreciate it.

X

**Comment by GarrettGunnell** [comment]
Sorry I'm not really interested in anything related to AI.

**Comment by X8X-XZDX** [comment]
I'm not working on a model or anything.  All I am doing is working on an interface.  It'd be just a few minutes of your time.

I would like to use what you have created here to model what an idea physically looks like.  To me, what I saw in your video was ways to handle vector data, how GPU's manipulate data, all honestly things I do not know about and am currently studying as a student, same as you.

What my goal is, is to create a new way to work with AI to produce..... something.  I can get into more specifics about art and expression and employment and all that later if you want, we likely agree to a T.

TLDR. I think we are using AI wrong, and we're gunna fuck up bad.  Part of what I am working on needs to be modeled, and your project does that to a T.

I just wanted to talk to you about how the project itself works, and how I would add something to it.  If I would even be allowed to for my own demonstration in a video.

X
